<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QMFuncSettingsWidget</class>
 <widget class="QWidget" name="QMFuncSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Funktionen</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="documentMode">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Daten</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>20</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>20</number>
       </property>
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLineEdit" name="leFuncFilter">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>25</height>
              </size>
             </property>
             <property name="clearButtonEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="tbFuncFilter">
             <property name="minimumSize">
              <size>
               <width>25</width>
               <height>25</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Funktionen Filtern</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../rcs/icons.qrc">
               <normaloff>:/icons/icons/lc_recsearch.png</normaloff>:/icons/icons/lc_recsearch.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>16</width>
               <height>16</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QTableView" name="tvFunc">
             <property name="selectionMode">
              <enum>QAbstractItemView::SingleSelection</enum>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
             <property name="sortingEnabled">
              <bool>true</bool>
             </property>
             <attribute name="horizontalHeaderDefaultSectionSize">
              <number>150</number>
             </attribute>
             <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
              <bool>true</bool>
             </attribute>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>false</bool>
             </attribute>
             <attribute name="verticalHeaderVisible">
              <bool>false</bool>
             </attribute>
             <attribute name="verticalHeaderMinimumSectionSize">
              <number>25</number>
             </attribute>
             <attribute name="verticalHeaderDefaultSectionSize">
              <number>25</number>
             </attribute>
             <attribute name="verticalHeaderHighlightSections">
              <bool>false</bool>
             </attribute>
             <attribute name="verticalHeaderShowSortIndicator" stdset="0">
              <bool>false</bool>
             </attribute>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QToolButton" name="tbFuncAdd">
               <property name="minimumSize">
                <size>
                 <width>25</width>
                 <height>25</height>
                </size>
               </property>
               <property name="toolTip">
                <string>Hinzuf√ºgen einer neuen Funktion</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="../rcs/icons.qrc">
                 <normaloff>:/icons/icons/sc_add.png</normaloff>:/icons/icons/sc_add.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>16</width>
                 <height>16</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="tbFuncRemove">
               <property name="minimumSize">
                <size>
                 <width>25</width>
                 <height>25</height>
                </size>
               </property>
               <property name="toolTip">
                <string>Entfernen einer selektierten Funktion</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="../rcs/icons.qrc">
                 <normaloff>:/icons/icons/sc_delete.png</normaloff>:/icons/icons/sc_delete.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>16</width>
                 <height>16</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Gruppen</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>20</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>20</number>
       </property>
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QTableView" name="tvFuncGroups">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <property name="sortingEnabled">
            <bool>true</bool>
           </property>
           <attribute name="horizontalHeaderDefaultSectionSize">
            <number>150</number>
           </attribute>
           <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
            <bool>true</bool>
           </attribute>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderMinimumSectionSize">
            <number>25</number>
           </attribute>
           <attribute name="verticalHeaderDefaultSectionSize">
            <number>25</number>
           </attribute>
           <attribute name="verticalHeaderHighlightSections">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderShowSortIndicator" stdset="0">
            <bool>false</bool>
           </attribute>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QToolButton" name="tbFuncGroupsAdd">
             <property name="minimumSize">
              <size>
               <width>25</width>
               <height>25</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Hinzuf√ºgen einer neuen Funktionsgruppe</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../rcs/icons.qrc">
               <normaloff>:/icons/icons/sc_add.png</normaloff>:/icons/icons/sc_add.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>16</width>
               <height>16</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="tbFuncGroupsRemove">
             <property name="minimumSize">
              <size>
               <width>25</width>
               <height>25</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Entfernen einer selektierten Funktionsgruppe</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../rcs/icons.qrc">
               <normaloff>:/icons/icons/sc_delete.png</normaloff>:/icons/icons/sc_delete.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>16</width>
               <height>16</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../rcs/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>leFuncFilter</sender>
   <signal>editingFinished()</signal>
   <receiver>QMFuncSettingsWidget</receiver>
   <slot>filterFunc()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>366</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel">
     <x>384</x>
     <y>363</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbFuncAdd</sender>
   <signal>clicked()</signal>
   <receiver>QMFuncSettingsWidget</receiver>
   <slot>addFunc()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>747</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>384</x>
     <y>363</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbFuncFilter</sender>
   <signal>clicked()</signal>
   <receiver>QMFuncSettingsWidget</receiver>
   <slot>filterFunc()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>748</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>384</x>
     <y>363</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbFuncRemove</sender>
   <signal>clicked()</signal>
   <receiver>QMFuncSettingsWidget</receiver>
   <slot>removeFunc()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>747</x>
     <y>156</y>
    </hint>
    <hint type="destinationlabel">
     <x>384</x>
     <y>363</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbFuncGroupsAdd</sender>
   <signal>clicked()</signal>
   <receiver>QMFuncSettingsWidget</receiver>
   <slot>addFuncGroups()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>748</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>384</x>
     <y>363</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbFuncGroupsRemove</sender>
   <signal>clicked()</signal>
   <receiver>QMFuncSettingsWidget</receiver>
   <slot>removeFuncGroups()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>748</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>384</x>
     <y>363</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addFunc()</slot>
  <slot>removeFunc()</slot>
  <slot>revertFunc()</slot>
  <slot>addFuncGroups()</slot>
  <slot>removeFuncGroups()</slot>
  <slot>revertFuncGroups()</slot>
  <slot>filterFunc()</slot>
 </slots>
</ui>
